packages/lisk-transactions/fixtures/transactions.json

```diff
@@ -15,6 +15,7 @@
 		},
 		"signature":
 			"69d0c7bc50b82465e2b0885cebc422aa9cd575050dc89905e22a6e2cc88802935c6809a59a2daa04ca99623a6fef76b7d03215ed7f401b74ef5301b12bfe2002",
+		"signatures": [],
```

Comment:
does signature has to be empty array when instantiating or if it's undefined, it will be an empty array?